tool/jt.rb

```diff
@@ -2122,13 +2125,18 @@ def graph(*args)
       when '--describe'
         describe = true
       when '--'
-        seafoam_args = args.dup
-        args.clear
+        until args.empty? || args.first == '---'
```

Comment:
The `---` is getting really complicated (e.g. I can't understand that code anymore), and `cfg2asm` seems to have a single option (`--no-comments`).
I think either we don't support `cfg2asm` options at all, or we use an env var for that.